tests/CMakeLists.txt

```diff
@@ -29,7 +29,10 @@ if(Boost_FOUND)
     add_subdirectory(unit/interleave_info)
     add_subdirectory(unit/alignment)
     add_subdirectory(unit/queueing)
-    add_subdirectory(unit/lfp)
+    # lfp test directly uses MPI functions, disable it in case of dynamic MPI case
+    if(NOT CORENRN_ENABLE_DYNAMIC_MPI)
+      add_subdirectory(unit/lfp)
+    endif()
```

Comment:
what is the good way to put it back?

Reply:
Have to look into details.

Reply:
IMO it's fine to leave this test disabled when dynamic MPI is enabled for now.